tests/jsunit/generator_test.js

```diff
@@ -19,10 +19,169 @@
  */
 'use strict';
 
+goog.require('Blockly.Dart');
+goog.require('Blockly.JavaScript');
+goog.require('Blockly.Lua');
+goog.require('Blockly.PHP');
+goog.require('Blockly.Python');
+
+var workspace;
+
+function defineGeneratorTestBlocks() {
+  Blockly.defineBlocksWithJsonArray([{
+    "type": "stack_block",
+    "message0": "",
+    "previousStatement": null,
+    "nextStatement": null
+  },
+  {
+    "type": "row_block",
+    "message0": "%1",
+    "args0": [
+      {
+        "type": "input_value",
+        "name": "INPUT"
+      }
+    ],
+    "output": null,
+    "nextStatement": null
+  },
+  {
+    "type": "controls_repeat_ext",
+    "message0": "Repeat Loop",
+    "message1": "%1",
+    "args1": [{
+      "type": "input_statement",
+      "name": "DO"
+    }],
+    "previousStatement": null,
+    "nextStatement": null
+  }]);
+}
+
+function undefineTestBlocks() {
```

Comment:
Name this `undefineGeneratorTestBlocks`.  Otherwise it (unfortunately) conflicts with a previous declaration of `undefineTestBlocks` in block_test.js